frontend/app.ts

```diff
@@ -0,0 +1,7 @@
+const sayHello = (name: string) => {
+    return `Hello, ${name}!`
+}
+
+let user = 'Lana';
+
+alert(sayHello(user));
```

Comment:
try to avoid "alerts", they block the execution of the program. more environmentally friendly to use the console log